### PR TITLE
Billing

### PR DESCRIPTION
**Pull request by yogirackspace**
Adding support for publishing atom entry when invoices are ready.

### CODE REVIEW

**Comment by shintasmith** [comment]
ok to test

File: sample_product_schemas/billing-transaction.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        Specifies the events related to Billing Account Transactions.
+    </description>
+    <attribute name="ran" type="string" use="required" minLength="1" maxLength="255">
+        Specifies the Billing Account Number.
+    </attribute>
+    <attribute name="tranType" type="string" use="required"  allowedValues="INVOICE">
+        Specifies the Type of Transaction
+    </attribute>  
+    <attribute name="tranRefNumber" type="string" use="required" minLength="1" maxLength="255">
+        Specifies the Billing Account Number.
```
Comment (shintasmith) [inline sample_product_schemas/billing-transaction.xml]: What is the difference between 'ran' and 'tranRefNumber'? They both say Billing Account Number.

Comment (yogirackspace) [inline sample_product_schemas/billing-transaction.xml]: My Bad. tranRefNumber is Transaction Reference Number.

**Comment by shintasmith** [comment]
@yogirackspace, your new resource needs to be added to the allfeeds.wadl. Here's a reference to the publisher's guide:
http://docs-internal.rackspace.com/cloud-feeds/api/v1.0/feeds-publishers-guide/content/Using_Cloud_Feeds.html

Also, we need to know when you need this change in Staging and Production environment, so we can plan our release.

**Comment by yogirackspace** [comment]
Since the resource is already existing I thought that is not needed..
Existing entry in allfeeds.wadl
       <resource id="billing_events"
                  path="billing/events" 
                  type="wadl/feed.wadl#AtomFeed wadl/product.wadl#BillingService”/>

 Let is know when your next standard release is. The teams have begin coding and I could communicate.

**Comment by shintasmith** [comment]
Ah, you are right. Yes, it will get picked up and you don't need to add it again to allfeeds.wadl.

Our standard weekly release is today (Mondays) to go to our Test environment. Then Staging on Thursdays and Production the following Tuesdays.

**Comment by yogirackspace** [comment]
That should be good. So would this pull also go today?

**Comment by shintasmith** [comment]
Let me check our QE load. From dev point of view, this can go to today's release.

**Comment by shintasmith** [comment]
QE gives a go-ahead. I'm merging it in

**Comment by yogirackspace** [comment]
Thanks Shinta and team!

**Pull request merged**